src/Illuminate/Auth/AuthManager.php

```diff
@@ -246,7 +246,7 @@ public function viaRequest($driver, callable $callback)
      *
      * @return \Closure
```

Comment:
You could narrow this down further:
```suggestion
     * @return \Closure(): \Illuminate\Contracts\Auth\Authenticatable|null
```